sims/quick_plot.py

```diff
@@ -134,14 +141,15 @@
 ax_fwdpp.legend(loc='upper left',frameon=False)
 ax_fwdpp.set_title("fwdpy11 with neutral mutations", fontsize='medium')
 ax_fwdpp_arg.set_title("fwdpy11 with pedigree tracking", fontsize='medium')
-ax_simupop.set_ylabel("Run time (hours)")
+#ax_simupop.set_ylabel("Run time (hours)")
 ax_fwdpp.set_ylabel("Run time (hours)")
-ax_simupop.set_title("simuPOP with neutral mutations", fontsize='medium')
-ax_simupop_arg.set_title("simuPOP with pedigree tracking", fontsize='medium')
-ax_simupop.set_xlabel('Scaled recombination rate (' + r'$\rho = 4Nr$)')
-ax_simupop_arg.set_xlabel('Scaled recombination rate (' + r'$\rho = 4Nr$)')
-ax_simupop_arg.set_xticks([1e3, 1e4, 1e5])
-for ax in (ax_simupop, ax_simupop_arg):
+#ax_simupop.set_title("simuPOP with neutral mutations", fontsize='medium')
+#ax_simupop_arg.set_title("simuPOP with pedigree tracking", fontsize='medium')
+#ax_simupop.set_xlabel('Scaled recombination rate (' + r'$\rho = 4Nr$)')
+#ax_simupop_arg.set_xlabel('Scaled recombination rate (' + r'$\rho = 4Nr$)')
+#ax_simupop_arg.set_xticks([1e3, 1e4, 1e5])
+for ax in (ax_fwdpp, ax_fwdpp_arg):
+    ax.set_xlabel('Scaled recombination rate (' + r'$\rho = 4Nr$)')
```

Comment:
revert?

Reply:
w/o it there is no label

Reply:
just meant revert the is part of the diff, lines 144-152

Reply:
That is not at all what GH is showing me in the browser.